Exercise.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DAY5_P1_EXERCISE_H
+#define DAY5_P1_EXERCISE_H
+
+#include <vector>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include <iostream>
+
+class Exercise {
+private:
+    std::string import_file{};
+    std::vector<char>files{};
+public:
+  bool read_file_string();
```

Comment:
make this whole thing a utility set. no need for member variables
```suggestion
  std::optional<std::string> read_file_string();
```